docs/docs/resources/pipeline-components/kafka-app.yaml

```diff
@@ -18,14 +18,18 @@
   to:
     topics: # required
       ${pipeline_name}-output-topic:
-        type: output # required
-        # role: topic-role # only used if type is `extra`
+        type: output
+      ${pipeline_name}-extra-topic:
+        # type: extra # Implied when role is specified
+        role: topic-role # only used if type is `extra`
```

Comment:
should we remove the comment now? it reads a little bit odd with the new line above

Reply:
yeah, but it must be reworded somehow to make it clear that role should not be defined for any type other than extra

Reply:
done